maunal/guides/software_templates.md

```diff
@@ -0,0 +1,135 @@
+they create software projects.
+
+# Existing Templates
+There are some open source projects that provide high quality templates
+that can be useful in software development. If your project falls under the 
+range of capabilities they have you can save a lot of time by using them or 
+adapting them to start your project.
+
+## Shablona
+[Shablona](https://github.com/uwescience/shablona) is a template repository that 
+is designed for scientific python projects. The template comes with everything a 
+small-scale project might need, and can be easily adapted for python projects.
+
+To get a sense of whether or not using the Shablona template is appropriate for 
+your project, checkout the 
+[example](https://github.com/uwescience/shablona/tree/master/examples) provided.
+
+## Cookiecutter
+[Cookiecutter](https://github.com/cookiecutter/cookiecutter) is not a template 
+repository like Shablona, instead it is a command-line utility that
+allows you to build a repository from a "pantry" of templates. This project has 
+an extensive 
+[Read the Docs](https://cookiecutter.readthedocs.io/en/1.7.2/index.html), and 
+it's definitely worth a review before you decide. 
+
+Additional documents that will give you a sense of whether or not to use this 
+utility:
+1. [Getting to Know Cookiecutter](https://cookiecutter.readthedocs.io/en/1.7.2/tutorial1.html) 
+2. [Usage](https://cookiecutter.readthedocs.io/en/1.7.2/usage.html)
+3. [Building a Cookiecutter](https://cookiecutter.readthedocs.io/en/1.7.2/first_steps.html)
+
+# Software Project Parts
+Even if neither of the above templates fit your requirements, there are still 
+some pieces that might benefit your project. JOSS (The Journal of Open Source 
+Software) has 
+a useful [review checklist](https://joss.readthedocs.io/en/latest/review_checklist.html) 
+for reviewers that you should review before beginning your project. 
+
+## Make Files
+Makefile is a scripting tool that
+compiles or re-compiles a program or process through a user-defined command in the command-line: ``make``. Makefiles streamline the build process for project contributors or reviewers.
+
+#### Resources:
+- [Introduction](https://www.gnu.org/software/make/manual/html_node/Introduction.html)
+- [Writing Makefiles](https://www.gnu.org/software/make/manual/html_node/Makefiles.html#Makefiles)
+- [Tutorial](https://makefiletutorial.com/)
+
+## CMake Files
+CMake generates makefiles, similar Make or a makefile, except that it is much 
+more compatible with C++.
+
+## Snakemake Files
+Snakemake is a workflow engine with a python-based language and an optimizing 
+execution environment. It allows developers to write human-readable workflows that 
+document themselves.
+
+### Resources:
+- [Tutorial](https://snakemake.readthedocs.io/en/stable/tutorial/tutorial.html)
+- [Examples](https://github.com/snakemake/snakemake/tree/master/examples)
+- [Wiki](https://snakemake.readthedocs.io/en/stable/index.html)
+
+#### Resources:
```

Comment:
This looks like CMake resources, so they might appear right after CMake Files section

Reply:
Yes they were, I'm not sure how they ended up there. Thank you